torchtune/modules/peft/lora.py

```diff
@@ -142,3 +298,29 @@ def _lora_b_init_params(x: nn.Linear) -> None:
     Initialize LoRA B weight to zeros.
     """
     nn.init.zeros_(x.weight)
+
+
+def _swap_lora_linear_with_qat(
```

Comment:
Personally I'd put this somewhere else, either in quantization.py or peft/_utils.py